cunumeric/deferred.py

```diff
@@ -191,6 +193,8 @@ def min_identity(ty):
     UnaryRedCode.ARGMIN: lambda ty: (np.iinfo(np.int64).min, min_identity(ty)),
     UnaryRedCode.CONTAINS: lambda _: False,
     UnaryRedCode.COUNT_NONZERO: lambda _: 0,
+    UnaryRedCode.ALL: lambda _: 1,
+    UnaryRedCode.ANY: lambda _: 0,
```

Comment:
Pedantic, but should this be True and False?

Reply:
Then should we chose it for `SUM`, `PROD`, and `COUNT_NONZERO`  to `True` and `False` instead of `1` and `0`?

Reply:
No, the identity elements for those operations should be numeric.

Reply:
This is pedantic, since in Python `1` will work in place of `True`, and `0` in place of `False`.

Reply:
OK